### ISSUE TITLE
Output path should respect the ember-cli environment

### ISSUE THREAD

**Issue by blkeller** [wackerservices/SitemapAutogenerator#2]
The add-on should take the `ember-cli` [environment](https://ember-cli.com/user-guide/#Environments) into consideration when determining the path under which to dump `sitemap.xml`.

For the **development** and **production** environments, `dist/` is the default, but in the **test** environment (which is the default environment when running `ember test`) the build product is placed in `tmp/class-tests_dist-[RANDOM_STRING].tmp/` by default.

Right now, the add-on uses `dist/` even for **test**, which means the build product getting tested is missing its `sitemap.xml`.  More importantly a potentially unrelated previous build stored in `dist/` will have its `sitemap.xml` clobbered by the one the add-on intended to produce for the **test** build.

**Comment by rupdegraff** [comment]
This issue has been addressed in commit 02add168bff041f7ba0aac0e60b28c5c2c5a3dd4 and the following `package.json` code replaces the previous method of calling `sitemap-autogenerator` from `ember-cli-build.js`

## Usage

  Add the following code to **package.json**: 
```js
"postbuild": "node -e \"require('./node_modules/sitemap-autogenerator/blueprints/sitemap-autogenerator/index').triggerSitemapBuilder('<YOUR SITE ROOT URL>')\"",
```

  Be sure to not have a trailing `/` after `<YOUR SITE ROOT URL>` or else you will get double `//` in your generated **sitemap.xml**
  
  `sitemap-autogenerator` will run at the end of each Ember build, which are run with: `npm build`

**Comment by blkeller** [comment]
That's great.  Thanks!
